framework/PageTable/PageTable.tsx

```diff
@@ -164,43 +165,29 @@ export function PageTable<T extends object>(props: PageTableProps<T>) {
   if (error) {
     return (
       <div className="dark-2" style={{ height: '100%' }}>
-        <EmptyState variant={EmptyStateVariant.small} style={{ paddingTop: 48 }}>
-          <EmptyStateIcon
-            icon={ExclamationCircleIcon}
-            color="var(--pf-global--danger-color--100)"
-          />
-          <Title headingLevel="h2" size="lg">
-            {/* Unable to connect */}
-            {props.errorStateTitle}
-          </Title>
-          {/* <EmptyStateBody>There was an error retrieving data. Check your connection and reload the page.</EmptyStateBody> */}
-          <EmptyStateBody>{error.message}</EmptyStateBody>
-        </EmptyState>
+        <EmptyStateError message={error.message} />
```

Comment:
@ShaiahWren I think it makes sense to allow `title` to be an optional param as well. WDYT?

Reply:
Sure, I can do that.

Reply:
Ah, so title is already an optional param called 'message' - in `EmptyStateError` the default title is `const title = t('Something went wrong');`.  Perhaps `error.message` is not the correct alternative to the default message. I'm trying to reconcile what is suggested in the mockups with what we already have:
<img width="563" alt="Screenshot 2023-03-29 at 9 12 49 AM" src="https://user-images.githubusercontent.com/64337863/228549797-9fdf0a1e-3b21-46fc-84b4-1d3a006be379.png">